packages/core/lib/commands/install.js

```diff
@@ -1,35 +1,54 @@
+const { callbackify } = require("util");
+
 const command = {
   command: "install",
   description: "Install a package from the Ethereum Package Registry",
   builder: {},
   help: {
-    usage: "truffle install <package_name>[@<version>]",
+    usage: "truffle install <package-identifier> [--alias]",
     options: [
       {
-        option: "package_name",
-        description:
-          "Name of the package as listed in the Ethereum Package Registry. (required)"
+        option: "packageIdentifier",
```

Comment:
Same as the above comment

Reply:
:+1: I've made this change - but just want to note that `packageIdentifier` was suggested above by @gnidan.

Reply:
Can you still make this change? It should be snake case.